backend/src/main/java/de/neuefische/timemanagement/backend/service/TaskService.java

```diff
@@ -5,15 +5,29 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
+import java.util.InputMismatchException;
+import java.util.InvalidPropertiesFormatException;
 import java.util.List;
 
 @Service
 @RequiredArgsConstructor
 public class TaskService {
 
     private final TaskRepo taskRepo;
+    private final IdService idService;
 
     public List<Task> getAllTasks(){
         return taskRepo.getAllTasks();
     }
+
+    public Task addTask(Task newTask){
+        if(newTask.title()==null ||newTask.title().equals("") ||newTask.dateTime()==null){
+            throw new IllegalArgumentException("missing title or date");
```

Comment:
maybe you should handle this Exception with a catch and response Information about the Exception to your frontend.